diff --git "a/src\\rules\\node\\file-extension-in-import.d.ts" "b/src\\rules\\node\\file-extension-in-import.error.d.ts"
index a8ad3a4..407a97c 100644
--- "a/src\\rules\\node\\file-extension-in-import.d.ts"
+++ "b/src\\rules\\node\\file-extension-in-import.error.d.ts"
@@ -12,7 +12,7 @@ export type FileExtensionInImportRule = {
       'always' | 'never',
       {
         tryExtensions?: string[];
-        [k: string]: 'always' | 'never';
+        [ext: `.${string}`]: 'always' | 'never';
       },
     ]
   >;