diff --git "a/src\\rules\\unicorn\\prefer-node-protocol.d.ts" "b/src\\rules\\unicorn\\prefer-node-protocol.error.d.ts"
index 32f1bd8..6a08e89 100644
--- "a/src\\rules\\unicorn\\prefer-node-protocol.d.ts"
+++ "b/src\\rules\\unicorn\\prefer-node-protocol.error.d.ts"
@@ -4,7 +4,17 @@ export type PreferNodeProtocolRule = {
   /**
    * Prefer using the `node:` protocol when importing Node.js builtin modules.
    *
-   * @see [prefer-node-protocol](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v46.0.0/docs/rules/prefer-node-protocol.md)
+   * @see [prefer-node-protocol](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v44.0.2/docs/rules/prefer-node-protocol.md)
    */
-  'unicorn/prefer-node-protocol': Rule<[RuleLevel]>;
+  'unicorn/prefer-node-protocol': Rule<
+    [
+      RuleLevel,
+      {
+        /**
+         * @deprecated
+         */
+        checkRequire?: boolean;
+      },
+    ]
+  >;
 };