diff --git "a/src\\rules\\graphql-eslint\\naming-convention.d.ts" "b/src\\rules\\graphql-eslint\\naming-convention.error.d.ts"
index 66dfc1e..c7a219d 100644
--- "a/src\\rules\\graphql-eslint\\naming-convention.d.ts"
+++ "b/src\\rules\\graphql-eslint\\naming-convention.error.d.ts"
@@ -84,8 +84,7 @@ export type NamingConventionRule = {
               VariableDefinition?: AsString | AsObject;
               allowLeadingUnderscore?: boolean;
               allowTrailingUnderscore?: boolean;
-              /**
-               */
+            } & {
               [k: string]: AsString | AsObject;
             },
           ]