diff --git "a/src\\rules\\typescript-eslint\\ban-ts-comment.d.ts" "b/src\\rules\\typescript-eslint\\ban-ts-comment.error.d.ts"
index 084c85f..2b0737e 100644
--- "a/src\\rules\\typescript-eslint\\ban-ts-comment.d.ts"
+++ "b/src\\rules\\typescript-eslint\\ban-ts-comment.error.d.ts"
@@ -6,5 +6,28 @@ export type BanTsCommentRule = {
    *
    * @see [ban-ts-comment](https://typescript-eslint.io/rules/ban-ts-comment)
    */
-  '@typescript-eslint/ban-ts-comment': Rule<[RuleLevel, any[]]>;
+  '@typescript-eslint/ban-ts-comment': Rule<
+    [
+      RuleLevel,
+      {
+        'ts-expect-error'?:
+          | boolean
+          | 'allow-with-description'
+          | { descriptionFormat?: string };
+        'ts-ignore'?:
+          | boolean
+          | 'allow-with-description'
+          | { descriptionFormat?: string };
+        'ts-nocheck'?:
+          | boolean
+          | 'allow-with-description'
+          | { descriptionFormat?: string };
+        'ts-check'?:
+          | boolean
+          | 'allow-with-description'
+          | { descriptionFormat?: string };
+        minimumDescriptionLength?: number;
+      },
+    ]
+  >;
 };