diff --git "a/src\\rules\\typescript-eslint\\lines-between-class-members.d.ts" "b/src\\rules\\typescript-eslint\\lines-between-class-members.error.d.ts"
index 658d816..9d9f4d3 100644
--- "a/src\\rules\\typescript-eslint\\lines-between-class-members.d.ts"
+++ "b/src\\rules\\typescript-eslint\\lines-between-class-members.error.d.ts"
@@ -9,8 +9,10 @@ export type LinesBetweenClassMembersRule = {
   '@typescript-eslint/lines-between-class-members': Rule<
     [
       RuleLevel,
+      'always' | 'never',
       {
-        [k: string]: any;
+        exceptAfterSingleLine?: boolean;
+        exceptAfterOverload?: boolean;
       },
     ]
   >;