diff --git a/src/rules/react/jsx-props-no-spreading.d.ts b/src/rules/react/jsx-props-no-spreading.d.ts
index 24b4d8d..7b02d0e 100644
--- a/src/rules/react/jsx-props-no-spreading.d.ts
+++ b/src/rules/react/jsx-props-no-spreading.d.ts
@@ -14,8 +14,6 @@ export interface JsxPropsNoSpreadingRule {
         custom?: 'enforce' | 'ignore';
         exceptions?: string[];
         [k: string]: any;
-      } & {
-        [k: string]: any;
       },
     ]
   >;
