diff --git a/src/rules/react/jsx-no-constructed-context-values.d.ts b/src/rules/react/jsx-no-constructed-context-values.d.ts
index b3617f5..1e67dc2 100644
--- a/src/rules/react/jsx-no-constructed-context-values.d.ts
+++ b/src/rules/react/jsx-no-constructed-context-values.d.ts
@@ -6,12 +6,5 @@ export interface JsxNoConstructedContextValuesRule {
    *
    * @see [jsx-no-constructed-context-values](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-no-constructed-context-values.md)
    */
-  'react/jsx-no-constructed-context-values': Rule<
-    [
-      RuleLevel,
-      ...{
-        [k: string]: any;
-      },
-    ]
-  >;
+  'react/jsx-no-constructed-context-values': Rule<[RuleLevel]>;
 }
