diff --git a/src/rules/react/jsx-props-no-spreading.d.ts b/src/rules/react/jsx-props-no-spreading.d.ts
index 24b4d8d..f4984af 100644
--- a/src/rules/react/jsx-props-no-spreading.d.ts
+++ b/src/rules/react/jsx-props-no-spreading.d.ts
@@ -13,9 +13,6 @@ export interface JsxPropsNoSpreadingRule {
         html?: 'enforce' | 'ignore';
         custom?: 'enforce' | 'ignore';
         exceptions?: string[];
-        [k: string]: any;
-      } & {
-        [k: string]: any;
       },
     ]
   >;
